python/s3_cross_account_access.py

```diff
@@ -0,0 +1,377 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+

```

Comment:
Please add the Gherkhin - template there: https://github.com/awslabs/aws-config-rules/blob/master/python/CLOUDTRAIL_ENABLED_V2/CLOUDTRAIL_ENABLED_V2.py